src/main/java/ioiobagiety/controller/HelloIOBagietyController.java

```diff
@@ -0,0 +1,50 @@
+package ioiobagiety.controller;
+
+
+import com.google.gson.Gson;
+import ioiobagiety.model.HelloIOBagiety;
+import ioiobagiety.service.HelloIOBagietyService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value="/bagieta")
+public class HelloIOBagietyController  {
+
+    @Autowired
+    private HelloIOBagietyService helloIOBagietyService;
+
+    @Autowired
+    private Gson gson;
+
+
+    @RequestMapping(method = RequestMethod.GET)
+    public ResponseEntity<String> getHelloIOBagiety() {
+
+        List<HelloIOBagiety> helloIOBagieties = helloIOBagietyService.getAll();
+
+        if (helloIOBagieties.isEmpty())
```

Comment:
Java code convention says that you should use braces even in one-line if statements